### ISSUE TITLE
current vs stale projects

### ISSUE THREAD

**Issue by jbothma** [impronunciable/hackdash#189]
We'd like a way to see which projects are current and which are not worked on at the moment.

We run regular meetups and try and match people to projects with a little "standup" meeting each time we meet. But project leads don't necessarily show up each week. But they might be available on slack.

We also don't want total newbies to spend a lot of time reading up on a project where they'll need lots of support getting going if the project is actually not active any more.

We have been using https://docs.google.com/document/u/1/d/1pVL-wjN6UVnJOkyqlV_SdvIbBihmapibRdKR4ZWbweA/edit until COVID hit and our meetups went online and numbers dwindled.

We'd like to rekindle things. I reckon if project leads get an email once every week or two, where they get a secret link that they have to click to indicate that the project is still live, then we can indicate to people visiting the dashboard which projects are active.

We'd still show the "stale" ones, but we'd just manage expectations.

e.g. the dashboard could have a section "active projects" at the top, and "inactive projects" at the bottom. Most of these projects are inactive, but at least one is still actually moving forward now and then and needs help.
![Screenshot_2021-05-29_22-52-32](https://user-images.githubusercontent.com/235801/120084478-bfe33080-c0d0-11eb-998b-5c3830854df2.png)

**Comment by jbothma** [comment]
what are your thoughts on this? Perhaps just a couple of UI tweaks and a cron job to mail project leads once every two weeks.

**Comment by impronunciable** [comment]
The email could be an opt-in setting, it could be spammy for people that don't want to use the feature.

**Comment by jbothma** [comment]
Yeah - I think most importantly opt-in at a board level because for our board we'd like all projects marked inactive if they're not actually being actively developed so that the small number of active projects can be found very easily.

In future, if there's demand, it could also be made opt-in per project - someone might like their own projects checked this way - even if it wasn't opted into on the board level.
